src/popup/components/Protocol/index.tsx

```diff
@@ -47,7 +53,7 @@ function ProtocolState() {
       await getProtocolOptIn().optIn(mnemonic);
       serviceOptedIn.reload();
       completedLiveness.reload();
-
+      getMnemonicSave().setMnemonic(mnemonic);
```

Comment:
Setting this state here would still have the problem I mentioned with reloading chrome/the extension. Either pass the mnemonic to the component, or write it to local storage (probably with `setCheckNeeded(mnemonic)`).